src/renderers/__tests__/EventPluginHub-test.js

```
@@ -32,6 +36,14 @@ describe('EventPluginHub', () => {
     }).toThrowError(
       'Expected onClick listener to be a function, instead got type string',
     );
+    if (ReactDOMFeatureFlags.useFiber) {
+      expectDev(console.error.calls.count()).toBe(1);
+      expectDev(console.error.calls.argsFor(0)[0]).toContain(
+        'Expected onClick listener to be a function, instead got type string',
+      );
+    } else {
+      expectDev(console.error.calls.count()).toBe(0);
```

This warning isn't included in stack since it fails early